# Designing Educational Experiences with Jupyter Notebooks:
##A variety of NCBI Computational Tools Distributed as Jupyter Notebooks
Poor documentation leads to poor understanding of a software. It is difficult to interpret other researchers' code without unequivocal documentation. Jupyter Notebook combines code and rich-text elements which allows the user to effectively learn, modify, and run the notebook.


# Why?
 
Disease surveillance research has led to the genome sequencing of many thousands of isolates. However, the annotation of these genome sequences is not straitgh forward and often does not provide researchers with a complete set of information. Here, we present a computational pipeline to compare de novo sequence contigs to the set of complete RefSeq genomes for i) determining appropriate reference genome for whole-genome alignment, ii) proving a clean list of genes and annoated proteinsand iii) and highlitghing sets of genes invlovled in antibiotic resistance and virulence. We demonstrate our pipeline using data from S. aureus as a paradigm, owing to its high sequence variability, and therefore less well-curated genomic sequences in public databases.

### Dependencies:
* Python 3 (packages such as BioPython, Macplotlib, Pandas)
* Jupyter Notebook


# Case Study
![My image](https://github.com/NCBI-Hackathons/NCBIJupyter/blob/master/case_study.png)

### Method:




### Project Team:
* Ray Anderson
* Richard Copin 
* Evgeny Ivanchenko
* Victor Joukov
* Lon Phan

## Example screenshots
List of genes
![My image](https://github.com/NCBI-Hackathons/NCBIJupyter/blob/master/gene_list.png)






